dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -1110,4 +1109,8 @@ public long sum() {
     @JsonInclude(Include.NON_EMPTY)
     public static class EmptyImageType implements OneOfFileWrapperImageType {
     }
+
+    public record EntryTypeDAOAndStats(String trsClassName, EntryDAO<? extends Entry<?, ?>> dao, Long numEntries) {
```

Comment:
If this record is meant for use solely within `ToolsApiServiceImpl`, recommend that we change the access to something less public, so we don't accidentally couple other code to it.  Similar thoughts about `GitReferenceInfo` above.

Reply:
Don't feel strongly about this, but agreed it does seem unlikely so changing